### PR TITLE
Files for Midtrans Review:  21Dec

### PR DESCRIPTION
**Pull request by rashmikampurath**
@rizdaprasetya  I have updated the files with all the feedbacks. I request you to kindly go through it.

### CODE REVIEW

File: en/snap/interactive-demo.md [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
+- Backend: **NodeJS** (hosted on CodeSandbox)
+- Frontend: **HTML** and **JavaScript**
 
-You will see Snap Transaction Token as vizualization of API response on backend integration. Then you can click **"Proceed to Payment"** to test the frontend.
+## Usage Explanation
+You will see *Snap* Transaction Token as a visualization of API response on backend integration.
+  1. Click **Proceed to Payment** to test the frontend.
 
-?> Please wait until the window/iframe below is fully loaded. It may take some time as it try to build the backend.
+  > **Note**: Please wait until the window/iframe below is fully loaded. It may take some time while it tries to build the backend.
```
Comment (rashmikampurath) [inline en/snap/interactive-demo.md:29]: As per the team discussion, the "Notes" inside the numbered steps use Blockquotes. Thus, this change is incorporated.

File: en/snap/interactive-demo.md [hunk 2]
```diff
@@ -53,38 +54,37 @@ script.setAttribute('data-element-id','snap-backend-demo');
...
+## Testing Payment
+You can perform successful transaction by entering the card credentials given below.
 
 Name | Value
 --- | ---
 Card Number | `4811 1111 1111 1114`
 CVV | `123`
-Exp Month | Any month (e.g: `02`)
-Exp Year | Any future year (e.g: `2025`)
+Exp Month | Any month in MM format. For example, `02`
```
Comment (rashmikampurath) [inline en/snap/interactive-demo.md:80]: The period from the table item is removed.

File: en/snap/interactive-demo.md [hunk 3]
```diff
@@ -53,38 +54,37 @@ script.setAttribute('data-element-id','snap-backend-demo');
...
-Now that you understand the basic of how Snap integration works, this is just demonstration of **Step 1 & Step 2** [explained in the guide](/en/snap/integration-guide.md?id=integration-steps). Please follow the next step from there.
+For more test payment credentials, refer to [Testing Payments on Sandbox](/en/technical-reference/sandbox-test.md).
 
 <div class="my-card">
 
 #### [Next Step](/en/snap/integration-guide.md?id=_4-handle-after-payment)
-Follow Snap integration Guide next steps, to proceed with complete integration.
-</div>
\ No newline at end of file
+This was a demonstration of Step 1 and Step 2 on [Snap Integration Guide](/en/snap/integration-guide.md?id=integration-steps). Please follow the next steps given in the [Snap Integration Guide](/en/snap/integration-guide.md?id=integration-steps) to proceed with the complete integration.
```
Comment (rashmikampurath) [inline en/snap/interactive-demo.md:89]: The broken link is fixed here.

File: en/snap/with-plugins.md [hunk 4]
```diff
@@ -56,269 +60,298 @@ Here is the list of Content Management System plugins and extensions that are su
...
 | Payment Notification URL | [your-site-url]/snap/payment/notification |
-| Finish Redirect URL | [your-site-url]/snap/index/finish |
-| Error Redirect URL | [your-site-url]/snap/index/finish |
-| Unfinish Redirect URL | [your-site-url]/snap/index/finish |
+| Finish Redirect URL      | [your-site-url]/snap/index/finish         |
+| Error Redirect URL       | [your-site-url]/snap/index/finish         |
+| Unfinish Redirect URL    | [your-site-url]/snap/index/finish         |
 
-> **Note:**
+> **Note**:
```
Comment (rashmikampurath) [inline en/snap/with-plugins.md:327]: Notes with multiple lines appear like shown below

File: en/snap/with-plugins.md [hunk 5]
```diff
@@ -350,68 +382,87 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
...
+7. Find the **Midtrans Pay** module in the module manager and click **Configure**. Configure *Midtrans Pay* page is displayed. Follow the steps given below.
+    * Enter **Payment Option Display Text**. This text appears on the button displayed to the customer.
+    * In the **Environment** list, click the appropriate environment. *Development* for testing transaction, *Production* for real transaction.
+    * Enter **Merchant ID**.
+    * Enter **Client key.**
+    * Enter **Server key**.
+    * Select desired order status for successful payments, from **Map payment SUCCESS status to** list.
+    * Select desired order status for payment failure, from **Map payment FAILURE status to** list.
+    * Select desired order status for challenge payment, from **Map payment PENDING/CHALLENGE status to** list.
+     > **Note**: Other fields are optional. You may leave it as is.
```
Comment (rashmikampurath) [inline en/snap/with-plugins.md:413]: Notes with one line appear as shown in the image below.

File: en/snap/with-plugins.md [hunk 6]
```diff
@@ -420,226 +471,272 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
...
+## OpenCart
+Midtrans ❤️ OpenCart! This is official Midtrans extension for the OpenCart E-commerce platform.
 
-#### Requirements:
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* OpenCart minimal 2.0 or later **|** Tested up to v3.0
+* PHP version 5.4 or later
+* MySQL version 5.0 or later
+* Midtrans plugin for OpenCart: For OpenCart v3.0 [[GitHub](https://github.com/Midtrans/Midtrans-Opencart3) | [Zip](https://github.com/Midtrans/Midtrans-Opencart3/archive/master.zip) ]; For OpenCart v2.3 [ [GitHub](https://github.com/Midtrans/SNAP-Opencart-2.3/) | [Zip](https://github.com/Midtrans/SNAP-Opencart-2.3/archive/master.zip)]; For OpenCart v2.o, OpenCart v2.1, or OpenCart v2.2 [ [GitHub](https://github.com/veritrans/SNAP-Opencart) | [Zip](https://github.com/veritrans/SNAP-Opencart/archive/master.zip)]
```
Comment (rizdaprasetya) [inline en/snap/with-plugins.md:482]: Screen Shot 2020-12-22 at 12 09 24
That is a long and ugly line. The visual of the original (with sub-bullet-point) is better:

Screen Shot 2020-12-22 at 12 09 13

please fix

Also, `v2.o`??? careful please, it is`v2.0`

File: en/snap/with-plugins.md [hunk 7]
```diff
@@ -56,269 +60,298 @@ Here is the list of Content Management System plugins and extensions that are su
...
-* PHP version v5.4 or greater
-* MySQL version v5.0 or greater
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* WordPress v3.9 or later **|** Tested up to v5.x
+* WooCommerce v2 or later **|** Tested up to v3.5.2
+* PHP version v5.4 or later
+* MySQL version v5.0 or later
 * PHP CURL enabled server/host
+* Midtrans plugin for WooCommerce [ [GitHub](https://github.com/veritrans/SNAP-Woocommerce) | [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip) ]
```
Comment (rizdaprasetya) [inline en/snap/with-plugins.md:80]: I think we should improve this to be more intuitive for reader. Please change this part of download requirement (applies to all CMS) into:

> * Download Midtrans plugin for XYZ: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))

And for those with multiple links:

> * Download Midtrans plugin for XYZ (please choose accordingly): 
>   * For XYZ v1.x: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))
>   * For XYZ v2.x: [Zip file](https://example.org) (Open sourced on [GitHub](https://example.org))

Thanks

Comment (rashmikampurath) [reply en/snap/with-plugins.md:80]: Ok. I will do that

**Pull request merged**